### DISCUSSION TITLE
Why won't my computer screen display the images I tell it to???

### DISCUSSION THREAD

**Discussion by sophief10** [jspsych/jsPsych#2157] [Q&A] [answered]
Hi everyone! I am new to jsPsych and programming in general, and am having difficulty using the "preload" plugin. 

I would like to display an image that is saved on my computer under the file path "C:\Users\myName\OneDrive\Pictures\experiment\red.jpg."

I have tried the following codes to no success:

### Code 1
```
var pre_task = {
    type: 'survey-likert',
    questions: [
        {
            prompt: <img src="C:\Users\myName\OneDrive\Pictures\experiment\red.jpg"></img><p>I enjoy looking at this image.</p>',
            labels: scale_1
        }
    ]
};
```

**Result:** A webpage that displays the appropriate text and Likert scale, but an icon in place of the image (see attached screenshot). 
<img width="774" alt="2021-09-15 (3)" src="https://user-images.githubusercontent.com/90805854/133526736-71694312-f418-4fcc-ac6f-69e7c6ea6e68.png">

### Code 2
```
var preload = {
    type: 'preload',
    images: ['C:\Users\myName\OneDrive\Pictures\experiment\red.jpg']
};

var pre_task = {
    type: 'survey-likert',
    questions: [
        {
            prompt: <img src="C:\Users\myName\OneDrive\Pictures\experiment\red.jpg"></img><p>I enjoy looking at this image.</p>',
            labels: scale_1
        }
    ]
};
```

**Result:** A webpage that reads "The experiment failed to load" (see attached screenshot).
<img width="745" alt="2021-09-15 (2)" src="https://user-images.githubusercontent.com/90805854/133526699-b7c9a692-4752-4241-849c-4d7ba555ef2b.png">

If anyone has an idea as to why my code is not working, would you please share your insights? Again, I am new to programming and am struggling to find the answer myself. 

Thank you for reading this post!

**Comment by mary-9555** [comment, marked as answer]
Hey there!
I'm also new to programming and am trying to make an online task for the first time. Also I'm not a native english spoken (so sorry if its not fluent!)
As no one yet answered this, Here is my suggestions (which I hope be correct :)!):
1. Make sure your images' folder is in the Experiment folder. You can make a folder titled "images" (for example) in the experiment folder, and put all your images in it.
2. You don't need to write the drive's name, I think this one is the correct way (if you do the example i mentioned above):
prompt: &lt;img src="images\red.jpg">&lt;/img>&lt;p>I enjoy looking at this image.</p&gt;'

**Reply by sophief10** [reply]
Thank you! I carefully went over my folder/image names like you suggested and realized the problem: I forgot to add a quotation mark to the front of "<img>...". *facepalm*

Thanks for your help!
